UnitTests/PayPalTests/PayPalClient_Tests.swift

```diff
@@ -0,0 +1,31 @@
+import XCTest
+import PayPalCheckout
+@testable import PaymentsCore
+@testable import PayPal
+
+final class PayPalClient_Tests: XCTestCase {
+
+    func testPayPalClient_whenPayPalCheckoutOnCancelInvoked_completionCalledWithCancellationState() throws {
```

Comment:
This should be a test for the `start` method instead, not the internal `setupPayPalCheckoutConfig` method.

Also since the file is already for `PayPalClient` the test method name can be more specific to the method we're testing. Example: `testStart_when....`

Reply:
About this and https://github.com/paypal/iOS-SDK/pull/25#discussion_r740329241
I'm not sure what is a good way to unit test `start`, since `Checkout.start()` is an UI experience + the flow involves network requests + user actions required.
So I decided to split the callbacks into another function `setupPayPalCheckoutConfig`. imo from our standpoint we just wanna test our callbacks being invoked when PayPalCheckout's callbacks are invoked.

Reply:
I agree with @scannillo it'd be more coverage complete if we can test the `start` method.